src/content/learn/conditional-rendering.md

```diff
@@ -703,46 +703,38 @@ function Drink({ name }) {
 export default function DrinkList() {
   return (
     <div>
-      <Drink name="tea" />
-      <Drink name="coffee" />
+      <Drink name="čaj" />
+      <Drink name="kafa" />
     </div>
   );
 }
 ```
 
 </Sandpack>
 
-Here the information about each drink is grouped together instead of being spread across multiple conditions. This makes it easier to add more drinks in the future.
+Ovde su informacije o svakom piću grupisane umesto da se nalaze u više uslova. Ovo olakšava dodavanje drugih pića u budućnosti.
 
-Another solution would be to remove the condition altogether by moving the information into objects:
+Drugo rešenje bi bilo potpuno uklanjanje uslova pomeranjem informacija u objekte:
 
 <Sandpack>
 
 ```js
-const drinks = {
-  tea: {
-    part: 'leaf',
-    caffeine: '15–70 mg/cup',
-    age: '4,000+ years'
-  },
-  coffee: {
-    part: 'bean',
-    caffeine: '80–185 mg/cup',
-    age: '1,000+ years'
-  }
-};
+const drinks = [
+  {name: 'čaj', part: 'list', caffeine: '15–70 mg/šolja', age: '4,000+ godina'},
```

Comment:
In order to properly translate drink name, I needed to introduce new field, `name`. I've also created PR on the main react repo in order to simplify translation of object property values for others as well: https://github.com/reactjs/react.dev/pull/7296

Reply:
Got it!